src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20200430091144_EES775UpdateContact.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES775UpdateContact : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.UpdateData(
+                table: "Contacts",
+                keyColumn: "Id",
+                keyValue: new Guid("ee8b0c92-b556-4670-904b-c265f0332a9e"),
+                columns: new[] { "ContactName", "ContactTelNo" },
+                values: new object[] { "Daisy Astill", "07741 118332" });
```

Comment:
Isn't it weird that we're putting people's actual contact details into the migrations?

Reply:
We need to get rid of all of this content seed data as it should never have been here to begin with. While it's here though I think it's safer to update via this mechanism, otherwise someone could make a change here in future and not realise that it's already diverged in Production.

Also we need a feature for managing these content updates so they aren't change requests!